ui/inspectors/system-section.reel/language-and-region.reel/language-and-region.js

```diff
@@ -46,12 +107,44 @@ exports.LanguageAndRegion = Component.specialize(/** @lends LanguageAndRegion# *
                 Promise.all(loadingPromises).then(function() {
                     this.isLoading = false;
                 });
+                var today = new Date();
+                this.dateFormatShortOptions = this._generateDateFormatConvertedList(today, this.shortDateFormats);
+                this.dateFormatMediumOptions = this._generateDateFormatConvertedList(today, this.mediumDateFormats);
+                this.dateFormatLongOptions = this._generateDateFormatConvertedList(today, this.longDateFormats);
+                this.dateFormatFullOptions = this._generateDateFormatConvertedList(today, this.fullDateFormats);
+                this.timeFormatShortOptions = this._generateDateFormatConvertedList(today, this.shortTimeFormats);
+                this.timeFormatMediumOptions = this._generateDateFormatConvertedList(today, this.mediumTimeFormats);
+                this.timeFormatLongOptions = this._generateDateFormatConvertedList(today, this.longTimeFormats);
+            }
+        }
+    },
+
+    _generateDateFormatConvertedList: {
+        value: function(today, dateOptionList) {
+            var _formattedDateList = [];
```

Comment:
no need to put an underscore at the beginning of a variable name, it is only used to distinguish private properties.